### ISSUE TITLE
can we add some mentors to solve any problem related to these stacks ??

### ISSUE THREAD

**Issue by Rupeshiya** [dwyl/learn-html5#39]
@nelsonic  If we do this it will be very helpful for learner .

**Comment by nelsonic** [comment]
@Rupeshiya Yeah, we need a "lower friction" way of asking and answering questions (_in general_).
(_I'm **always** happy to **mentor** people who have specific questions_)

I don't know the "_solution_" to this ...
But I do know that if we can "_solve_" it, we will ["_win the internet_"](https://www.google.com/search?q=win+the+internet&tbm=isch).

**Comment by Rupeshiya** [comment]
@nelsonic  If you will be starting it ,I will be happy to join you since I love to help others.
Hope we are going to make it happen.

**Comment by iteles** [comment]
@Rupeshiya @nelsonic Through the medium of github, the way we could do this is to add a section at the top of the repo reminding people that if they need help they should open an issue.

If we continue to build a community around dwyl repos like this one, those who want to mentor others will begin to answer the question in the repos that they can and we can keep building on the amazing work @karinakozarova has done so far with the `readme` for this repo!
